### ISSUE TITLE
add Single Cell Portal as source in catalogue

### ISSUE THREAD

**Issue by amnonkhen** [ebi-ait/dcp-ingest-central#484]
[Single Cell Portal](https://singlecell.broadinstitute.org/single_cell)

Mentioned by @tburdett on Slack on 12/10/2021.

**Comment by amnonkhen** [comment]
@gabsie does this look OK?

![image.png](https://images.zenhubusercontent.com/600ed297091d3ea152b83c2d/84ae6f20-6048-4852-8a3f-476854cb4f9d)

**Comment by gabsie** [comment]
Hi, @amnonkhen @tburdett 

I don't think what you are suggesting above is yet true - as far as I know the wranglers are not consulting the Single Cell portal or looking for projects there.

Is that the task? For us to start importing and triaging from there as well? That's a nice idea. Then I suggest that we insert this in the list when it happens.

@tburdett it would be nice to know how important this is and if politically relevant.
Otherwise we can put it in the radar of the Ops team

Gabs

**Comment by tburdett** [comment]
@gabsie I completely agree that we should coordinate the update to the docs and the roll out of checks (i.e. update the docs when we really start importing and triaging from the single cell portal)

This came from a conversation with Tim; improving links between the different initiatives affiliated with the DCP is important, although we don't need to switch focus to deliver this, it can be done when we're next reviewing our processes.

**Comment by ofanobilbao** [comment]
This is on hold for now
